src/java/picard/sam/markduplicates/util/OpticalDuplicateFinder.java

```diff
@@ -72,9 +72,12 @@ public OpticalDuplicateFinder(final String readNameRegex, final int opticalDupli
      * unmodified a copy of the list should be passed to this method.
      *
      * @param list a list of reads that are determined to be duplicates of one another
+     * @param keeper a single PhysicalLocation that is the one being kept as non-duplicate, and thus should never be
```

Comment:
I keep thinking of sports analogies for `keeper`, how about `representative` instead like in htsjdk's [DuplicateSet](https://github.com/samtools/htsjdk/blob/master/src/java/htsjdk/samtools/DuplicateSet.java)?